File: thegrey/The_skin.pl
sub EVENT_SPAWN {
  $x = $npc->GetX();
  $y = $npc->GetY();
  $z = $npc->GetZ();
  quest::set_proximity($x - 30, $x + 30, $y - 30, $y + 30);
}

sub EVENT_ENTER {
  quest::emote("around your eyes tightens in reaction to the vacuum.");
}

File: thegrey/The_skin.lua
  
function event_spawn(e)
	local ex = e.self:GetX();
	local ey = e.self:GetY();
	local ez = e.self:GetZ();
	eq.set_proximity(ex - 30, ex + 30, ey - 30, ey + 30, ez - 30, ez + 30);
end

function event_enter(e)
	e.self:Emote("around your eyes tightens in reaction to the vacuum.");
	eq.depop_with_timer();
end
